src/main/java/EchoServer.java

```diff
@@ -10,7 +11,13 @@ public EchoServer(SocketManager serverSocketManager) {
     public void start(int port) throws IOException {
         serverSocketManager.connect(port);
         String clientInput = serverSocketManager.receiveString();
-        serverSocketManager.present(Message.confirmClientMessageReceived());
-        serverSocketManager.present(clientInput);
+        while(!clientInput.equals(COMMAND_TO_QUIT)) {
```

Comment:
Should there be some doc in the command explaining to hit q to quit?

Reply:
@ssunday should I document this in the README?

Reply:
That would be a good first place, but when a user launches the program how would they know? Is the expectation they'd always read the readme?

Reply:
like how would the end user know?

Help menu?

Reply:
Understood!